### PR TITLE
Remove outdated input file generisdata_nur_ren.prn

### PR DESCRIPTION
**Pull request by lecfab**
## Purpose of this PR
See @robertpietzcker's comment https://github.com/remindmodel/remind/pull/1782#discussion_r1768592480: The values in `generisdata_nur_ren.prn` are outdated and overwritten in files `f_maxProdGradeRegi*.cs3r`. This PR removes the outdated file,

## Type of change

(Make sure to delete from the Type-of-change list the items not relevant to your PR)

- [ ] Bug fix 
- [x] Refactoring
- [ ] New feature 
- [x] Minor change (default scenarios show only small differences)
- [ ] Fundamental change
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the [coding etiquette](https://github.com/remindmodel/remind/blob/develop/main.gms#L80)
- [x] I performed a self-review of my own code
- [x] I explained my changes within the PR, particularly in hard-to-understand areas
- [x] I checked that the [in-code documentation](https://github.com/remindmodel/remind/blob/develop/main.gms#L120) is up-to-date
- [ ] I adjusted the reporting in [`remind2`](https://github.com/pik-piam/remind2) where it was needed
- [ ] I adjusted `forbiddenColumnNames` in [readCheckScenarioConfig.R](https://github.com/remindmodel/remind/blob/develop/scripts/start/readCheckScenarioConfig.R) in case the PR leads to deprecated switches
- [x] All automated model tests pass (`FAIL 0` in the output of `make test`)
- [ ] The changelog `CHANGELOG.md` [has been updated correctly](https://gitlab.pik-potsdam.de/rse/rsewiki/-/wikis/Standards-for-Writing-a-Changelog)

## Further information (optional):

* Test runs are here: `/p/tmp/fabricel/nurPrnRemoval-remind`
* Comparison of results (what changes by this PR?): nothing changes, see `/p/tmp/fabricel/nurPrnRemoval-remind/compScen-nurPrnRemoval-2024-09-25_20.51.30-H12.pdf`

### CODE REVIEW

File: modules/32_power/IntC/equations.gms [hunk 1]
```diff
@@ -68,39 +69,37 @@ q32_limitCapTeStor(t,regi,teStor)$( t.val ge 2020 ) ..
   =l=
   sum(te2rlf(teStor,rlf),
     vm_capFac(t,regi,teStor)
-  * pm_dataren(regi,"nur",rlf,teStor)
```
Comment (lecfab) [inline modules/32_power/IntC/equations.gms:71]: Value here was always 1 for storage technologies. I think it is find to remove it.

File: modules/32_power/IntC/equations.gms [hunk 2]
```diff
@@ -68,39 +69,37 @@ q32_limitCapTeStor(t,regi,teStor)$( t.val ge 2020 ) ..
...
-*' Require a certain capacity  of either hydrogen or gas turbines as peaking backup capacity. The driver is the testor capacity, which in turn is determined by v32_storloss 
+*** H2 turbines are not needed if sufficient gas turbines (ngt) are available to provide flexibility. 
+*' Require a certain capacity of either hydrogen or gas turbines as peaking backup capacity. The driver is the teStor capacity, which in turn is determined by v32_storloss 
 q32_h2turbVREcapfromTestor(t,regi)..
   vm_cap(t,regi,"h2turbVRE","1")
   + vm_cap(t,regi,"ngt","1")
   =g=
-  sum(teStor, 
-    p32_storageCap(teStor,"h2turbVREcapratio") * vm_cap(t,regi,teStor,"1") )
+  sum(teStor, p32_storageCap(teStor,"h2turbVREcapratio") * vm_cap(t,regi,teStor,"1") )
```
Comment (robertpietzcker) [inline modules/32_power/IntC/equations.gms:88]: Hm, I do think it is clearer if the parts "what set elements does the sum go over" and "what values are summed" are split over two lines.
This is also the way most sums are implemented, so I would disagree with changing this default. 

If you see it differently, let's maybe discuss offline?

File: modules/32_power/IntC/equations.gms [hunk 3]
```diff
@@ -68,39 +69,37 @@ q32_limitCapTeStor(t,regi,teStor)$( t.val ge 2020 ) ..
...
 ;
 
 *** h2turbVRE hydrogen turbines should only be built in conjunction with storage capacities and not on its own
 q32_h2turbVREcapfromTestorUp(t,regi)..
   vm_cap(t,regi,"h2turbVRE","1")
   =l=
-  sum(te$teStor(te), 
-      p32_storageCap(te,"h2turbVREcapratio") * vm_cap(t,regi,te,"1") )
+  sum(teStor, p32_storageCap(teStor,"h2turbVREcapratio") * vm_cap(t,regi,teStor,"1") )
```
Comment (robertpietzcker) [inline modules/32_power/IntC/equations.gms:95]: as above

File: core/datainput.gms [hunk 4]
```diff
@@ -961,8 +958,11 @@ $if %cm_MAgPIE_coupling% == "on" p_efFossilFuelExtr(regi,"pebiolc","n2obio") = 0
 
 display p_efFossilFuelExtr;
 
-pm_dataren(regi,"nur",rlf,te)     = f_datarenglob("nur",rlf,te);
-pm_dataren(regi,"maxprod",rlf,te) = sm_EJ_2_TWa * f_datarenglob("maxprod",rlf,te);
+*** capacity factors (nur) are 1 by default
+pm_dataren(regi,"nur",rlf,te)     = 1;
+
+*** value copied from old file generisdata_nur_ren.prn (50EJ = 1.5855TWa)
```
Comment (robertpietzcker) [inline core/datainput.gms:964]: maybe even more explicitly: 
*** the only RE technology that doesn't get maxprod and nur from f_maxProdGradeRegixxx files are geohe and geohdr, which take their values from the old file generisdata_nur_ren.prn

File: modules/32_power/IntC/equations.gms (hunk 2 above)
Comment (lecfab) [reply modules/32_power/IntC/equations.gms]: I've seen both, but don't mind either way!
All 3 teStor sums are now over 2 lines.

File: core/datainput.gms (hunk 4 above)
Comment (lecfab) [reply core/datainput.gms:964]: 50EJ right?
Much better with these explanations thanks.

**Review by robertpietzcker — APPROVED** [review]

**Pull request merged**